plugin/storage/grpc/config/config.go

```diff
@@ -15,24 +15,32 @@
 package config
 
 import (
+	"context"
 	"fmt"
 	"os/exec"
 	"runtime"
+	"time"
 
 	"github.com/grpc-ecosystem/grpc-opentracing/go/otgrpc"
 	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/go-plugin"
 	"github.com/opentracing/opentracing-go"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 
 	"github.com/jaegertracing/jaeger/plugin/storage/grpc/shared"
 )
 
 // Configuration describes the options to customize the storage behavior.
 type Configuration struct {
-	PluginBinary            string `yaml:"binary" mapstructure:"binary"`
-	PluginConfigurationFile string `yaml:"configuration-file" mapstructure:"configuration_file"`
-	PluginLogLevel          string `yaml:"log-level" mapstructure:"log_level"`
+	PluginBinary             string        `yaml:"binary" mapstructure:"binary"`
+	PluginConfigurationFile  string        `yaml:"configuration-file" mapstructure:"configuration_file"`
+	PluginLogLevel           string        `yaml:"log-level" mapstructure:"log_level"`
+	RemoteServerAddr         string        `yaml:"server" mapstructure:"server"`
+	RemoteTLS                bool          `yaml:"tls" mapstructure:"tls"`
+	RemoteCAFile             string        `yaml:"cafile" mapstructure:"cafile"`
```

Comment:
We have a shared config/options struct for configuring TLS, see other storage configs for examples